crmprtd/process.py

```diff
@@ -57,14 +56,26 @@ def add_process_args(parser):  # pragma: no cover
         "--start_date",
         help="Optional start time to use for processing "
         "(interpreted with dateutil.parser.parse).",
-        default=str(datetime.min),
     )
     parser.add_argument(
         "-E",
         "--end_date",
         help="Optional end time to use for processing "
         "(interpreted with dateutil.parser.parse).",
-        default=str(datetime.max),
+    )
+    parser.add_argument(
```

Comment:
Good idea, though I don't think that you have clearly articulated in the help strings how this option interacts with `start_date` and `end_date` and which takes precedence. The somewhat convoluted logic below, is indicative of that lack of clarity.